integration-cli/docker_cli_daemon_test.go

```diff
@@ -326,7 +326,7 @@ func (s *DockerDaemonSuite) TestDaemonIptablesCreate(c *check.C) {
 	}
 
 	// make sure the container is not running
-	runningOut, err := s.d.Cmd("inspect", "--format='{{.State.Running}}'", "top")
```

Comment:
Yup, we made this change to a bunch of tests as we ported things to spf13/pflags.  The old pkg/mflag had special code that was used exclusively by tests to strip these single quotes.  Normally on a shell they would be removed by bash, but since this command isn't passed to bash they don't have any purpose.